tests/http2-frames/test.rules

```diff
@@ -0,0 +1 @@
+alert http2 any any -> any any (frame:http2.hdr; content:"|00 00 04 08 00|"; flow:to_server; sid:1;)
```

Comment:
Why don't we test all frame types?

Reply:
:-p

Because I tested the one I needed

Adding the other ones